models/zone/src/zone/zone-draft/presets/sample-data-goodstore/index.ts

```diff
@@ -0,0 +1,9 @@
+import countryGermany from './country-germany';
+import countryGreatBritain from './country-unitedkingdom';
+
+const presets = {
+  countryGermany,
+  countryGreatBritain,
```

Comment:
One last modification 😅

```suggestion
import countryUnitedKingdom from './country-unitedkingdom';

const presets = {
  countryGermany,
  countryUnitedKingdom,
```